alacritty_terminal/src/term/mod.rs

```diff
@@ -2252,6 +2270,10 @@ mod tests {
         }
     }
 
+    /// Test selection across lines
+    ///
+    ///
+    /// 1.  [a  ][a" ][a  ][a  ][a" ]
```

Comment:
this might be an incorrect picture of the line that's being tested.

Reply:
Not quite sure what this "picture" is supposed to mean.